sdk/ai/azure-ai-resources/azure/ai/resources/_index/_documents/document.py

```diff
@@ -127,7 +127,7 @@ def dumps(self) -> str:
         return json.dumps({"content": self.data, "metadata": self._metadata, "document_id": self.document_id})
 
     @classmethod
-    def loads(cls, data: str) -> "Document":
+    def loads(cls, data: str) -> "StaticDocument":
```

Comment:
StaticDocument looks better than Document, but I'm a bit curious what do we use it for?

Reply:
From reading the code, "StaticDocument" will hold information about user's files, like jsonl. We will perform index search on this files.